### ISSUE TITLE
tv3sport.dk

### ISSUE THREAD

**Issue by Henrik77** [reek/anti-adblock-killer#2940]
http://imgur.com/a/1xX0e
Google Chrome
Tampermonkey
AdBlock
EasyList + AakList

**Comment by uBlock-user** [comment]
try - `tv3sport.dk##.vjs-overlay-message-overlay`

**Comment by Henrik77** [comment]
Hi
Thanks for your advise.
I have tried "tv3sport.dk##.vjs-overlay-message-overlay" as customized filter in AdBlock - I assume this is where to ad it ?

Unfortunaltely it is still detecting, and bluring the video.

Any other suggestions ?

BR
  Henrik

**Comment by uBlock-user** [comment]
That's the Geo-Lock, not anti adblock. I'm also receiving that as I do not have an IP address within Denmark.

**Comment by Henrik77** [comment]
No - this is not due to IP location.

The text overlay on the video translated to English :
----
We se that you have an adblocker installet. 
We are depending on income from adds in order to continiue delivering
freed sports news.
The video will therefor be shown in its lowest quality.
Deactivate your adblocker to watch it in better quality.
----

And if i disable the adblocker i get 30 seconds of adds, and after that 
i can watch the video news in full quailty.

But it might not be easy to test in other countries than denmark, if they 
(as it sounds) also has an IP location check.

**Comment by uBlock-user** [comment]
I'm receiving a geo-lock. Mind posting a link ?

**Comment by Henrik77** [comment]
Hey

Link to a screenshot ?
Such I there in my first post : http://imgur.com/a/1xX0e

Or do you mean link to one of the videos i'm trying to watch - it could be : http://www.tv3sport.dk/sport/motorsport/nyheder/alonso-raser-over-mc-laren-racer-tidligere-formel-1-mester-eftersoger-fart

Thanks for supporting.

**Comment by uBlock-user** [comment]
> Or do you mean link to one of the videos i'm trying to watch - it could be : http://www.tv3sport.dk/sport/motorsport/nyheder/alonso-raser-over-mc-laren-racer-tidligere-formel-1-mester-eftersoger-fart

Yes, I meant that precisely. Always post a link when reporting, so it makes it easy for me to reproduce.

Secondly, after I add `tv3sport.dk##.vjs-overlay-message-overlay` I receive this Geo-Lock I mentioned from earlier -

![](https://i.gyazo.com/a9bf674e4cbb66df487b302aaeac5870.png)
